win32/Lib/postinstall/__main__.py

```diff
@@ -0,0 +1,3 @@
+from win32.lib.postinstall.pywin32_postinstall import main
```

Comment:
Do we really need the `postinstall` intermediate dir? Any reason not to just have pywin32_postinstall.py directly in win32/Lib?